lib/qesapdeployment.pm

```diff
@@ -1707,4 +1732,90 @@ sub qesap_export_instances {
     record_info('EXPORT', "SSH keys and instances data uploaded to test results:\n" . join("\n", @upload_files));
 }
 
+=head3 get_job_state
```

Comment:
As you prefer but this one was maybe good in sles4sap_publiccloud. If you think it is good here, please mind to rename it to `qesap_get_job_state`

Reply:
OK, I will!

Reply:
Done